requirements_all.txt

```diff
@@ -876,9 +876,11 @@ pencompy==0.0.3
 pexpect==4.6.0
 
 # homeassistant.components.rpi_pfio
+# homeassistant.components.rpi_pfio2
```

Comment:
There should not be two integrations that do the exact same thing. This should be in the same integration.